docs/azure_arc_jumpstart/azure_arc_k8s/day2/cluster_api/cluster_api_gitops_helm/_index.md

```diff
@@ -0,0 +1,229 @@
+
+- The GitOps configurations will create the namespace, deploy NGINX Ingress controller and Hello-Arc application
+
+- User is verifying the cluster and making sure the extension and GitOps configuration is deployed.
+
+- User is making an edit on the GitHub repo that will cause Flux GitOps to detect this change and trigger an update to the pod deployment.
+
+## Azure Arc Kubernetes GitOps Configuration
+
+To create the GitOps Configuration, we will use the _k8s-configuration flux create_ command while passing in values for the mandatory parameters. This scenario provides you with the automation to configure the GitOps on your Azure Arc-enabled Kubernetes cluster.
+
+> **Note: Before configuring the GitOps, make sure that the _kubectl_ context is pointing to your Azure Arc-enabled Kubernetes cluster. To do that, you can refer to the [official Kubernetes documentation](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) to find the options to change the kubecontext to different Kubernetes clusters.**
+
+![Screenshot showing current kubectl context pointing to CAPI cluster](./03.png)
+
+- In the screenshot below, notice how currently there is no GitOps configuration in your Arc-enabled Kubernetes cluster.
+
+    ![Screenshot showing Azure Portal with no Azure Arc-enabled Kubernetes GitOps configurations](./04.png)
+
+## Cluster-level Config vs. Namespace-level Config
+
+### Cluster-level Config
+
+With Cluster-level GitOps config, the goal is to have "horizontal components" or "management components" deployed on your Kubernetes cluster which will then be used by your applications. Good examples are Service Mesh, security products, monitoring solutions, etc. A very popular example will also be ingress controllers, which is exactly the Nginx ingress controller we will deploy in the next section.
+
+### Namespace-level Config
+
+With Namespace-level GitOps config, the goal is to have Kubernetes resources deployed only in the namespace selected. The most obvious use-case here is simply your application and its respective pods, services, ingress routes, etc. In the next section, will have the "Hello Arc" application deployed on a dedicated namespace.
+
+- Download [the script](https://github.com/microsoft/azure_arc/blob/main/azure_arc_k8s_jumpstart/cluster_api/gitops/helm/az_k8sconfig_helm_capi.sh) using below command
+
+    ```shell
+    curl -L https://raw.githubusercontent.com/microsoft/azure_arc/main/azure_arc_k8s_jumpstart/cluster_api/gitops/helm/az_k8sconfig_helm_capi.sh -o ~/az_k8sconfig_helm_capi.sh
+    ```
+
+- Edit the environment variables to match your environment parameters followed by running the _`. ./az_k8sconfig_helm_capi.sh`_ command.
+
+    ![Screenshot parameter](./05.png)
+
+    For example:
+
+    ![Screenshot parameter examples](./06.png)
```

Comment:
Make sure you are rotating secrets before we publish

Reply:
It is random secret values